db_sql.go

```diff
@@ -534,13 +632,19 @@ func SqlCreateSchema_Session(conx *DBConnection) error {
 	// we made some changes that are not backwards compatible, need to force a logout for all users
 	versions = append(versions, `DELETE FROM authsession;`)
 
+	versions = append(versions, `
+	DROP INDEX idx_authsession_identity;
+	ALTER TABLE authsession DROP COLUMN identity;
+	ALTER TABLE authsession ADD COLUMN userid BIGSERIAL;
+	ALTER TABLE authsession ADD CONSTRAINT fk_storeUserId_sessionUserId FOREIGN KEY (userid) REFERENCES userstore (userid);`)
```

Comment:
We should discuss the pros & cons of enabling these foreign key constraints. I'm not sure it's the right thing to do.
Also, there must be an explicit index on userid, so that we can efficiently wipe all sessions for a given user.